tests/Microsoft.Identity.Test.Integration.netcore/Infrastructure/ConfidentialAppSettings.cs

```diff
@@ -37,9 +37,9 @@ public class ConfidentialAppSettings
     {
         private class PublicCloudConfidentialAppSettings : IConfidentialAppSettings
         {
-            public string ClientId => UseAppIdUri? "https://request.msidlab.com" : "f62c5ae3-bf3a-4af5-afa8-a68b800396e9";
+            public string ClientId => UseAppIdUri? "https://request.msidlab.com" : "88f91eac-c606-4c67-a0e2-a5e8a186854f";
```

Comment:
this AppIdUri `"https://request.msidlab.com"` is for the lab app `f62c5ae3-bf3a-4af5-afa8-a68b800396e9` in MSFT tenant.

Reply:
the lab app's app id uri : `api://88f91eac-c606-4c67-a0e2-a5e8a186854f`

Reply:
What is the name of the cert for this app? LabVaultAccessCert? Does this change for all the apps for cca testing or only the ones using secret?

Reply:
I now added LabVaultAccessCert to this app

Reply:
updated